sample/sample-with-di/app/src/main/java/com/dropbox/kaiken/sample/advancedsample/app/AccountStore.kt

```diff
@@ -0,0 +1,32 @@
+package com.dropbox.kaiken.sample.advancedsample.app
+
+import com.dropbox.kaiken.sample.advancedsample.helloworldfeature.DiSampleUser
+import com.dropbox.kaiken.skeleton.scoping.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import javax.inject.Inject
+
+interface AccountStore {
```

Comment:
I would think this interface is in library code?

Reply:
I was thinking this is left up to the app, the only piece that we care about is a `Flow<Set<SkeletonUser>>`.  The rest would be up to the app - thoughts?

Reply:
I think including the interface would be minimal to method count and give folks a shape for making their account store/flow provider. Really its not an accountstore but a `AllUsersFlowProvider`. Its up to you I don't feel strongly.